src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,54 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  static addAnimal(animal) {
+    this.alive.push(animal);
+  }
+
+  static removeAnimal(animal) {
+    const indexOfAnimal = this.alive.indexOf(animal);
+
+    if (indexOfAnimal !== -1) {
+      this.alive.splice(indexOfAnimal, 1);
+    }
+  }
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+    Animal.addAnimal(this);
+  }
+
+  hide() {
+    this.hidden = this.hidden ? 0 : true;
+  }
```

Comment:
[Checklist](https://github.com/mate-academy/js_herbivores_and_carnivores/blob/master/checklist.md) the 3rd point